src/vm/wren_vm.c

```diff
@@ -1621,3 +1622,46 @@ void wrenGetVariable(WrenVM* vm, const char* module, const char* name,
   
   setSlot(vm, slot, moduleObj->variables.data[variableSlot]);
 }
+
+// Prints to WrenVM.errorFn if provided, otherwise prints to stderr.
+void wrenPrintError(WrenVM* vm, const char* format, ...)
+{
+  va_list args;
+  
+#if defined( _MSC_VER )
```

Comment:
I think this is a good call.

- [ ] Use vsnprintf if available – check `#if defined(_MSC_VER) && _MSC_VER < 1900`